test/celeritas/ext/RootImporter.test.cc

```diff
@@ -360,10 +360,18 @@ TEST_F(RootImporterTest, volumes)
     }
 
     const unsigned int expected_material_ids[] = {1, 1, 1, 1, 0};
-    const std::string  expected_names[]
-        = {"box", "boxReplica", "boxReplica", "boxReplica", "World"};
-    const std::string expected_solids[]
-        = {"box", "boxReplica", "boxReplica2", "boxReplica3", "World"};
+
+    const std::string expected_names[] = {"box0x125555be0",
+                                          "boxReplica0x125556d20",
+                                          "boxReplica0x125557160",
+                                          "boxReplica0x1255575a0",
+                                          "World0x125555f10"};
```

Comment:
(not related to this line but close by in the diff) I think your ROOT files still have all the embedded EM data?

For this line also: `static const char* const expected_names[]` is better practice (since the code as writen constructs and initializes a bunch of local std::strings from static text data).